index.bs

```diff
@@ -2506,14 +2511,22 @@ To <dfn>resolve the requested features</dfn> given |requiredFeatures| and |optio
     1. Return the [=tuple=] <code>(|consentRequired|, |consentOptional|, |granted|)</code>
   1. Add every [=feature descriptor=] in the [=default features=] table associated with |mode| to the indicated feature list if it is not already present.
   1. For each |feature| in |requiredFeatures| perform the following steps:
-    1. If |feature| is not a valid [=feature descriptor=], return <code>null</code>.
+    1. If the |feature| is <code>null</code>, [=continue=] to the next entry.
```

Comment:
Maybe the [language in the Permissions spec](https://www.w3.org/TR/permissions/#dom-permissions-query) is a bit more concise

Reply:
Hmmmm. This is tricky to do since we need to convert it to multiple things, potentially.